config/config_schema.json

```diff
@@ -405,23 +408,27 @@
             "description": "Path to CARTA backend executable",
             "type": "string",
             "examples": [
+                "usr/bin/carta_backend",
                 "/usr/local/bin/carta_backend"
-            ]
+            ],
+            "default": "/usr/bin/carta_backend"
         },
         "killCommand": {
             "description": "Path to CARTA kill script",
             "type": "string",
             "examples": [
-                "/usr/local/bin/carta_kill_script.sh"
-            ]
+                "/usr/local/bin/carta-kill-script"
+            ],
+            "default": "/usr/local/bin/carta-kill-script"
         },
         "rootFolderTemplate": {
             "description": "Top-level path of directories accessible to CARTA. The `{username}` placeholder will be replaced with the username",
             "type": "string",
             "examples": [
                 "/home/{username}",
                 "/"
-            ]
+            ],
+            "default": "/usr/share/carta"
```

Comment:
Just realised a problem: if the user installs the backend from source, the default image will be installed to `/usr/local/share/carta`. `/usr/share/carta` may not exist at all -- I think we need to check if it does and fall back to `local`.

I think this also means that we should backport the default image to the 1.4 release in the _source_, otherwise if someone builds the release backend from source they won't get the image at all and we may have a situation where neither directory exists.

Reply:
What _should_ happen if neither default location exists? This may happen if the backend is not installed at all or installed in a custom directory. I guess in the latter case the user has to configure the controller anyway in order to find the backend.

Reply:
ok, so we should skip the default value in the schema itself, and then check `/usr/share/carta` first, and then `/usr/local/share/carta` if no value is assigned. If neither location exists, the controller should bail out

Reply:
That makes sense. I'll do the backport in the backend. I'm putting this in the documentation as I go.

Reply:
(I think we should check only for the existence of the directory; checking for the default image as well would be too fiddly, and it may have been moved, etc..)